### PR TITLE
Added a pizza fact

### PR DESCRIPTION
**Pull request by david-001**
## Description
Added a pizza fact in the pizza-facts-&-trivia.md

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Relates to #36 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by CBID2 — CHANGES_REQUESTED** [review]
Thanks for your Pull Request @david-001! We've merged in some new PRs since you submitted. Can you merge the latest changes and resolve conflicts? If you need some help with that, check out this post: [Keeping Your Branch Up to Date and Handling Merge Conflicts While Waiting for PR Reviews](https://dev.to/opensauced/keeping-your-branch-up-to-date-and-handling-merge-conflicts-while-waiting-for-pr-reviews-3b3h). Also, can you fill out the description and other sections of your PR's form? It'll make it easier for me and other maintainers to review your contribution. Check out the blog post, [Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9) to learn how.

**Comment by david-001** [comment]
I merged the latest changes and resolved the conflicts. Can you specify which sections of my PR form I need to fill out? The sections I left out are either not relevant or optional. Thanks.

**Comment by CBID2** [comment]
> I merged the latest changes and resolved the conflicts. Can you specify which sections of my PR form I need to fill out? The sections I left out are either not relevant or optional. Thanks.

Fill out the `Description` section. In there,briefly describe the fact you add to the file.

**Comment by david-001** [comment]
Sorry about that. I filled out the Description.

File: pizza-facts-&-trivia.md
Comment (CBID2) [inline pizza-facts-&-trivia.md:21]:
```suggestion
11. There is a professional-level sporting competition dedicated to dough-spinning. At the World Pizza Championships, you'll face off against top-tier competitors in five intense events: Freestyle Acrobatic Dough Tossing, Largest Dough Stretch, Fastest Dough, Fastest Pizza Box Folding, and the Pizza Triathlon. Additionally, there's an elite Masters division specifically dedicated to Freestyle Acrobatic Dough Tossing.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @david-001. Your entry is off to a great start. Just found a typo that needs fixing.

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're close @david-001. Just need to make a few more adjustments.

**Comment by david-001** [comment]
I think I resolved the conflict this time.

**Review by CBID2 — APPROVED** [review]
Looks good to me

**Pull request merged**
